ansible_bender/__main__.py

```diff
@@ -0,0 +1,2 @@
+from cli import main
+main()
```

Comment:
Please add two empty lines after the imports section.
Also, add a condition to call main only if `__name__ == '__main__'`

Reply:
@webknjaz When I went through the python docs I found this, 
```
For a package, the same effect can be achieved by including a __main__.py module, 
the contents of which will be executed when the module is run with -m.
```
https://docs.python.org/3/library/__main__.html 
Please let me know your thoughts regarding this. I suppose the file name `__main__.py` does the job itself.

Reply:
yes, the best practise is still to enclose the `main()` call with `if __na...` as @webknjaz suggests, otherwise weird behavior can happen -- you can even see it in the example you link

Reply:
@TomasTomecek @webknjaz  thank you for the clarification will update it that way!